Merging internal commits for release/7.0

vseanreesermsft opened the pull request.

Commit:
Quarantine Settings_MaxHeaderListSize_Server (#45812)

dotnet-bot committed:
Merge in 'release/7.0' changes

dotnet-bot committed:
Merge in 'release/7.0' changes

dotnet-bot committed:
Merge in 'release/7.0' changes

Tratcher committed:
React to new HttpClient header size check #45811 (#45880)

dotnet-bot committed:
Merge in 'release/7.0' changes

dotnet-bot committed:
Merge in 'release/7.0' changes

dotnet-bot committed:
Merge in 'release/7.0' changes

dotnet-bot committed:
Merge in 'release/7.0' changes

dotnet-bot committed:
Merge in 'release/7.0' changes

dotnet-bot committed:
Merge in 'release/7.0' changes

dotnet-bot committed:
Merge in 'release/7.0' changes

dotnet-bot committed:
Merge in 'release/7.0' changes

dotnet-bot committed:
Merge in 'release/7.0' changes

dotnet-bot committed:
Merge in 'release/7.0' changes

dotnet-bot committed:
Merge in 'release/7.0' changes

dotnet-bot committed:
Merged PR 29038: [internal/release/7.0] Update dependencies from dnceng/internal/dotnet-runtime

This pull request updates the following dependencies

[marker]: <> (Begin:2802be6f-a6f7-477e-8165-08da7ef6284d)
## From https://dev.azure.com/dnceng/internal/_git/dotnet-runtime
- **Subscription**: 2802be6f-a6f7-477e-8165-08da7ef6284d
- **Build**: 20230119.10
- **Date Produced**: January 20, 2023 10:13:15 PM UTC
- **Commit**: 0a2bda10e81d901396c3cff95533529e3a93ad47
- **Branch**: refs/heads/internal/release/7.0

[DependencyUpdate]: <> (Begin)

- **Updates**:
  - **Microsoft.Extensions.Configuration.Binder**: [from 7.0.2 to 7.0.3][1]
  - **Microsoft.Extensions.Hosting**: [from 7.0.0 to 7.0.1][2]
  - **Microsoft.Extensions.Options**: [from 7.0.0 to 7.0.1][2]
  - **Microsoft.Internal.Runtime.AspNetCore.Transport**: [from 7.0.2-servicing.22606.5 to 7.0.3-servicing.23069.10][1]
  - **Microsoft.NET.Runtime.MonoAOTCompiler.Task**: [from 7.0.2 to 7.0.3][1]
  - **Microsoft.NET.Runtime.WebAssembly.Sdk**: [from 7.0.2 to 7.0.3][1]
  - **Microsoft.NETCore.App.Ref**: [from 7.0.2 to 7.0.3][1]
  - **Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.browser-wasm**: [from 7.0.2 to 7.0.3][1]
  - **Microsoft.NETCore.App.Runtime.win-x64**: [from 7.0.2 to 7.0.3][1]
  - **Microsoft.NETCore.BrowserDebugHost.Transport**: [from 7.0.2-servicing.22606.5 to 7.0.3-servicing.23069.10][1]
  - **System.Diagnostics.DiagnosticSource**: [from 7.0.0 to 7.0.1][2]
  - **System.Security.Cryptography.Pkcs**: [from 7.0.0 to 7.0.1][2]
  - **System.Text.Json**: [from 7.0.1 to 7.0.2][3]

[1]: https://dev.azure.com/dnceng/internal/_git/dotnet-runtime/branches?baseVersion=GCd037e07&targetVersion=GC0a2bda1&_a=files
[2]: https://dev.azure.com/dnceng/internal/_git/dotnet-runtime/branches?baseVersion=GCd099f07&targetVersion=GC0a2bda1&_a=files
[3]: https://dev.azure.com/dnceng/internal/_git/dotnet-runtime/branches?baseVersion=GC97203d3&targetVersion=GC0a2bda1&_a=files

[DependencyUpdate]: <> (End)

[marker]: <> (End:2802be6f-a6f7-477e-8165-08da7ef6284d)

dotnet-bot committed:
Merged PR 29263: [internal/release/7.0] Update dependencies from dnceng/internal/dotnet-efcore

This pull request updates the following dependencies

[marker]: <> (Begin:10156ce0-bf6e-4dac-1399-08da7ef5fa55)
## From https://dev.azure.com/dnceng/internal/_git/dotnet-efcore
- **Subscription**: 10156ce0-bf6e-4dac-1399-08da7ef5fa55
- **Build**: 20230113.2
- **Date Produced**: January 13, 2023 10:56:36 PM UTC
- **Commit**: 01a74ccf9db71c23785f6a6ace038fed2ee7ae41
- **Branch**: refs/heads/internal/release/7.0

[DependencyUpdate]: <> (Begin)

- **Updates**:
  - **dotnet-ef**: [from 7.0.2 to 7.0.3][1]
  - **Microsoft.EntityFrameworkCore**: [from 7.0.2 to 7.0.3][1]
  - **Microsoft.EntityFrameworkCore.Design**: [from 7.0.2 to 7.0.3][1]
  - **Microsoft.EntityFrameworkCore.InMemory**: [from 7.0.2 to 7.0.3][1]
  - **Microsoft.EntityFrameworkCore.Relational**: [from 7.0.2 to 7.0.3][1]
  - **Microsoft.EntityFrameworkCore.Sqlite**: [from 7.0.2 to 7.0.3][1]
  - **Microsoft.EntityFrameworkCore.SqlServer**: [from 7.0.2 to 7.0.3][1]
  - **Microsoft.EntityFrameworkCore.Tools**: [from 7.0.2 to 7.0.3][1]

[1]: https://dev.azure.com/dnceng/internal/_git/dotnet-efcore/branches?baseVersion=GC0c8605d&targetVersion=GC01a74cc&_a=files

[DependencyUpdate]: <> (End)

[marker]: <> (End:10156ce0-bf6e-4dac-1399-08da7ef5fa55)

vseanreesermsft committed:
Merge commit 'febee99db845fd8766a13bdb391a07c3ee90b4ba' into internal-merge-7.0-2023-02-14-1049

wtgodbe commented:
May be worth including https://github.com/dotnet/aspnetcore/pull/46251 in this PR

dougbu committed:
[release/7.0] Update baselines, et cetera
- update baselines to latest 7.0.x release
- enable baseline validation
- update the .NET SDK to the latest 7.0.1xx version
- update 6.0.x site extension version
